### PR TITLE
DE47637: LX > Spacing of Back to Content button on minibar

### PR DESCRIPTION
**Pull request by kristian-torres**
## Relevant Rally Links

[DE47637](https://rally1.rallydev.com/#/289692574792d/custom/355050439968?detail=%2Fdefect%2F628583327467&fdp=true) : LX > Spacing of Back to Content button on minibar

## Description

Problem - Current Behaviour

If the config variable d2l.Tools.Content.LearnerExperience.SideNavOpen is set to On (which results in the sidebar starting open), the spacing of the 'Back to Content' button and side navigation panel is wrong. If you close/reopen the side panel it is corrected.

## Goal/Solution

We fixed this in most scenarios in DE46051, looks like the one scenario of when the panel is already open needs to be fixed now.

## Video/Screenshots

https://user-images.githubusercontent.com/91631476/159953667-9f143d8e-ae30-4d40-8f31-20af579b94b4.mp4

## Remaining Work
- [x] @Kiefer-Co-D2L will dev test on his machine.

### CODE REVIEW

File: d2l-sequence-viewer/d2l-sequence-viewer.js [hunk 1]
```diff
@@ -523,8 +604,11 @@ class D2LSequenceViewer extends mixinBehaviors([
 		// Override the 'scroll' property set by the page to remove disabled scrollbar
 		document.body.style.overflow = 'hidden';
 
-		// Dynamically set sidebar left inner header width
-		this._sideBarSetInnerHeaderWidth();
+		const isSidebarContainerOffscreen = document.getElementsByTagName('d2l-sequence-viewer')[0].shadowRoot.querySelector('#sidebar-container').className.includes('prevent-offscreen-interaction');
+
+		if (!entity?.properties?.sideNavOpen && isSidebarContainerOffscreen) {
+			this._sideBarSetInnerHeaderWidth();
```
Comment (kristian-torres) [inline d2l-sequence-viewer/d2l-sequence-viewer.js:610]: explanation: this dynamic setting of the innner header width only occurs when the sideNav config is NOT set and when the sidebar container is "offscreen"

File: d2l-sequence-viewer/d2l-sequence-viewer.js [hunk 2]
```diff
@@ -35,6 +35,135 @@ class D2LSequenceViewer extends mixinBehaviors([
 	D2L.PolymerBehaviors.Siren.SirenActionBehaviorImpl,
 	D2L.PolymerBehaviors.Sequences.LocalizeBehavior,
 ], PolymerElement) {
```
Comment (kristian-torres) [inline d2l-sequence-viewer/d2l-sequence-viewer.js:37]: yeah... lint was complaining a LOT so I rearranged a few things :S haha

Comment (ianlaidlaw) [reply d2l-sequence-viewer/d2l-sequence-viewer.js:37]: that's my bad. when i did the introduction of Lit to this repo, some rules changed and now every file is complaining about fn order. i may put up a pr that just fixes all of them

**Review by Kiefer-Co-D2L — APPROVED** [review]
LGTM!  Dev tested successfully.

**Pull request merged**